components/Results/Results.tsx

```diff
@@ -0,0 +1,18 @@
+import {FunctionComponent} from "react";
+import ResultSettingArea from "./ResultSettingArea";
+
+interface Props {
+}
+
+const Results: FunctionComponent<Props> = ({}) => {
```

Comment:
`ResultSettingArea` seems to be unneeded here? We are not reusing it anywhere or extracting it out to simplify.